### PR TITLE
add function remove_lists

### PR DESCRIPTION
**Pull request by MarionBla**
The function is made to remove lists in columns with only single elements

### CODE REVIEW

File: fhir2dataset/query.py [hunk 1]
```diff
@@ -187,6 +188,23 @@ def _select_columns(self):
...
+    @timing
+    def _remove_lists(self):
+        """Remove lists from columns with only single elements
+        """
+        df = self.main_dataframe
+        list_col = []
+        for column in df.columns:
+            lengths = df[column].str.len()
+            if all(len < 2 for len in lengths):
+                list_col.append(column)
```
Comment (simonvadee) [inline fhir2dataset/query.py:200]: why don't you do
```
   df[column] = df[column].apply(lambda x: x[0])
```
directly here ?

Comment (simonvadee) [reply fhir2dataset/query.py:200]: you wouldn't need list_col at all

File: fhir2dataset/api_caller.py [hunk 2]
```diff
@@ -15,6 +15,8 @@
 
 def concat_cell(dataset, cols_name, element):
     # column of one row, this single cell is of the same type as element.value, i.e. a list
+    if element.value == []:
+        element.value = [None]
```
Comment (simonvadee) [inline fhir2dataset/api_caller.py:19]: why do you have to do that ? seems weird

File: fhir2dataset/query.py (hunk 1 above)
Comment (MarionBla) [reply fhir2dataset/query.py]: Yes it is much better thank you

File: fhir2dataset/query.py [hunk 3]
```diff
@@ -187,6 +188,18 @@ def _select_columns(self):
...
+    @timing
+    def _remove_lists(self):
+        """Remove lists from columns with only single elements
+        """
+        df = self.main_dataframe
+        for column in df.columns:
+            lengths = df[column].str.len()
+            if all(len < 2 for len in lengths):
+                df[column] = df[column].apply(lambda x: x[0])
+                logger.info(f"the column treated is {column}")
```
Comment (SaulLu) [inline fhir2dataset/query.py:200]: A small suggestion, here we could say 

```
logger.info(f"extraction of the unique element from the lists composing the {column} column")
```

Comment (MarionBla) [reply fhir2dataset/query.py]: Yes it is better, thank you for the review!

**Review by simonvadee — APPROVED** [review]

File: fhir2dataset/api_caller.py (hunk 2 above)
Comment (MarionBla) [reply fhir2dataset/api_caller.py:19]: every column is in a list and each element too, which explains the [[None]]

**Pull request merged**